thingsboard_gateway/connectors/bacnet/device.py

```diff
@@ -72,6 +74,19 @@ def __str__(self):
     def config(self):
         return self.__config
 
+    @property
+    def stopped(self):
+        return self.__stopped
+
+    @property
+    def reading_time(self):
+        return self.__reading_time
+
+    @reading_time.setter
+    def reading_time(self, new_reading_time):
+        self.__reading_time = new_reading_time
+        self.__poll_period += new_reading_time
```

Comment:
It looks wrong to increase poll_period on each assign. If reading time  more than poll period from config - we should assign reading time to poll period for next request.